tools/ios/build_xcframework.py

```diff
@@ -191,6 +204,15 @@ def parse_args():
         help="Specify a platform/arch pair to build. Repeat to specify multiple pairs. "
         "If no pairs are specified, all supported pairs will be built.",
     )
+    parser.add_argument(
+        "--cmake-extra-define",
+        "-D",
+        action="append",
+        default=[],
+        metavar="CMAKE_EXTRA_DEFINE",
+        dest="cmake_extra_defines",
+        help="Extra definition to pass to CMake (with the CMake -D option) during build system generation.",
+    )
```

Comment:
nit: Can this be the same as the ORT build.py for consistency? That allows multiple to be specified at once (e.g. `--cmake_extra_defines A=0 B=1` as well, and is called `cmake_extra_defines`. 

```
    parser.add_argument("--cmake_extra_defines", nargs="+", action="append", help="...",
```

Reply:
sure, updated it to be more consistent. using the name "--cmake-extra-defines" to be consistent with other options here